README.md

```diff
@@ -42,50 +42,59 @@ extension ImagePipeline {
 Let's start with the basics. Here's an example of how to use a new `RxNuke.Loading` protocol to load an image and display the result on success:
 
 ```swift
-ImagePipeline.shared.loadImage(with: url)
+ImagePipeline.shared.rx.loadImage(with: url)
     .subscribe(onSuccess: { imageView.image = $0.image })
     .disposed(by: disposeBag)
 ```
 
+Or
+
+```swift
+ImagePipeline.shared.rx.loadImage(with: url)
+    .bind(to: imageView.rx.image)
```

Comment:
Thanks for update the README! 

I'm not sure about bind usage here though. `loadImage(with:)` can complete with an error, and `bind` method doesn't like errors:

```swift
case let .error(error):
    rxFatalErrorInDebug("Binding error to publish relay: \(error)")
```

Reply:
The difference with subscribing is that we're subscribing for a "success" event only:

```swift
.subscribe(onSuccess: { imageView.image = $0.image })
```

This code is going to throw an fatal error if `loadImage(with:)` completes with an error:

```swift
.bind(to: imageView.rx.image)
```

We could use bind if we were handling errors in some way or another (e.g. ignoring them using `orEmpty`).

Reply:
I created #10 to fix this issue :)